provider/app.py

```diff
@@ -205,7 +226,8 @@ def main():
 
     # Make sure we log to the console
     streamHandler = logging.StreamHandler()
-    formatter = logging.Formatter('[%(asctime)s] [%(levelname)s] [??] [kafkatriggers] %(message)s')
+    formatter = logging.Formatter('[%(asctime)s] [%(levelname)s] '
```

Comment:
@bjustin-ibm we are adding python PEP8 linting for all python files over time; Adhering to the <80 char line limit will be mandatory.